apps/slides/admin.py

```diff
@@ -17,25 +17,25 @@ class SlideAdmin(admin.ModelAdmin):
 	form = SlideAdminForm
 	fields = (
 		'created',
-		'text', 
-		'slide_scheme', 
-		'color_bg', 
-		'color_font', 
-		'font', 
+		'text',
+		'slide_scheme',
+		'color_bg',
+		'color_font',
+		'font',
 		'font_size',
 		'display_start',
 		'display_end',
-		'location',
+		#'location',
```

Comment:
The code that has been commented out and is not used should be removed. Remove code throughout that has been commented out.

Reply:
removed commented lines